packages/app/src/cli/utilities/developer-platform-client/app-management-client.ts

```diff
@@ -199,23 +202,22 @@ export class AppManagementClient implements DeveloperPlatformClient {
     return (await this.session()).token
   }
 
+  async businessPlatformToken(): Promise<string> {
+    return (await this.session()).businessPlatformToken
+  }
+
   async refreshToken(): Promise<string> {
-    const {token} = await ensureAuthenticatedAppManagement([], process.env, {noPrompt: true})
+    const {appToken, businessPlatformToken} = await ensureAuthenticatedAppManagementAndBusinessPlatform(
+      [],
```

Comment:
Maybe worth changing the signature on the function, given how its being used? Looks like it _never_ actually cares about scopes or envs in actual usage, only prompt support. Should make this more readable.

Reply:
Good call, I've updated the function signature